static/app/views/userFeedback/index.spec.jsx

```diff
@@ -98,14 +94,14 @@ describe('UserFeedback', function () {
         orgId: organization.slug,
       },
     };
-    const wrapper = mountWithTheme(
+    render(
       <OrganizationContext.Provider value={organization}>
         <UserFeedback {...params} />
       </OrganizationContext.Provider>,
-      routerContext
+      {context: routerContext}
     );
 
-    expect(wrapper.find('UserFeedbackEmpty').prop('projectIds')).toEqual([]);
+    expect(screen.getByTestId('user-feedback-empty')).toBeInTheDocument();
```

Comment:
`findByTestId`? I think the test broke on one of these, just looking quickly

Reply:
nah it needs #39801

Reply:
left a related comment below